main.go

```diff
@@ -61,6 +63,20 @@ func main() {
 	}
 
 	if !config.Cfg.DeployedInHub {
+
+		cc, err := addonutils.NewConfigChecker("policy-spec-sync", config.Cfg.AggregatorConfigFile)
```

Comment:
Is `policy-spec-sync` correct here?

Reply:
Not final changes, Jorge. I am testing if it works.

Reply:
Sorry I reviewed too early. 😞 
Maybe the miscommunication happened because prow doesn't build draft PRs? I don't like that setting.

Reply:
No worries. Good to get an early review.